src/Domain/SmsDataReceivedPayload.php

```
@@ -0,0 +1,78 @@
+    private ?string $recipient;
+    private ?int $simNumber;
+    private string $data;
+    private string $receivedAt;
+
+    public function __construct(
+        string $messageId,
+        string $phoneNumber,
+        string $sender,
+        ?string $recipient = null,
+        ?int $simNumber = null,
+        string $data,
```

PHPStan (2.2.7)

[error] Deprecated in PHP 8.1: Required parameter $data follows optional parameter $simNumber.


(parameter.requiredAfterOptional)